### PR TITLE
Use Maven patch plugin to apply patches

### PR DESCRIPTION
**Pull request by heyagosper**

### CODE REVIEW

File: schemas/pom.xml [hunk 1]
```diff
@@ -59,6 +59,34 @@
...
+                        </goals>
+                        <configuration>
+                            <patchDirectory>src/main/patches/</patchDirectory>
+                            <targetDirectory>src/main/resources</targetDirectory>
+                            <patches>
+                                <patch>geodesyml.patch</patch>
+                            </patches>
+                            <optimizations>false</optimizations>
+                            <!--patchTrackingFile>${project.build.directory}/patch-log.txt</patchTrackingFile-->
+                            <naturalOrderProcessing>true</naturalOrderProcessing>
```
Comment (lbodor) [inline schemas/pom.xml]: Could we remove this comment please?

**Comment by oehm-smith** [comment]
:+1:  (looks good from what I understand)

**Comment by lbodor** [comment]
I've fixed the geodesyml-xsd.zip dependency problem. Could you please rebase and travis will re-run the build.

**Comment by lbodor** [comment]
👍

**Pull request merged**